plugin/al/translators/tests/testTranslators.py

```diff
@@ -440,8 +461,6 @@ def testNurbsCurve_PretearDownEditTargetWrite(self):
         self.assertEqual(cvcAttr.Get()[0], 10)
 
     def testMeshTranslator_multipleTranslations(self):
-        path = tempfile.NamedTemporaryFile(suffix=".usda", prefix="test_MeshTranslator_multipleTranslations_", delete=True)
```

Comment:
I think the importStageWithSphere call below is what creates the data used for this test.  I can't find a single use of this file anywhere, and the test does pass without it, so getting rid of it.